src/test/java/teammates/test/cases/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -106,88 +97,97 @@ private void testAddResponse() throws Exception {
         ______TS("test new response");
 
         submitPage.fillResponseTextBox(2, 0, "4");
-        submitPage.chooseMcqOption(3, 0, "It's good");
```

Comment:
Why this get deleted?

Reply:
You said to not to introduce dependency.
https://github.com/TEAMMATES/teammates/pull/8830/files/c63a9feb6c75d7b55ef78e78a6ef582bde268bce#diff-82bdb0dedfa18b6de94c0f00dda65ee9

Reply:
Ohhh. Sorry, I am doing review based on the changes compared to previous one :D